### PR TITLE
TSAPI-884 - Update API Docs for Jobcode Changes to Support Custom Fields

### PR DESCRIPTION
**Pull request by jsalazar7**
Added the customfields object to jobcodes.
Added the 'customfields' query parameter to GET /jobcodes that gates the retrieval of customfields.
Changed the wording of the required addon for custom fields from 'Advanced Tracking' to 'Custom Fields'. I could not find an addon named 'Advanced Tracking'.

### CODE REVIEW

**Review by elloboblanco — APPROVED** [review]
LGTM, please deploy + update & deploy private docs as well.

**Review by EldinTurulja — APPROVED** [review]
LGTM after merging 👍

**Pull request merged**
